pom.xml

```diff
@@ -27,22 +27,38 @@
       <dependency>
         <groupId>org.kohsuke</groupId>
         <artifactId>github-api</artifactId>
-        <version>1.95</version>
+        <version>1.99</version>
       </dependency>
       <dependency>
         <groupId>org.gitlab4j</groupId>
         <artifactId>gitlab4j-api</artifactId>
-        <version>4.11.15</version>
+        <version>4.12.17</version>
       </dependency>
       <dependency>
         <groupId>ch.qos.logback</groupId>
         <artifactId>logback-classic</artifactId>
         <version>1.2.3</version>
       </dependency>
+      <dependency>
+        <groupId>com.aliction</groupId>
+        <artifactId>bitbucket-client</artifactId>
+        <version>0.2.1</version>
+      </dependency>
+      <dependency>
+        <groupId>org.glassfish.jersey.media</groupId>
+        <artifactId>jersey-media-json-jackson</artifactId>
```

Comment:
why this dependency?

Reply:
For some reason the dependency is not picked from the main library, bitbucket-client, and without it I am getting Exception :
org.glassfish.jersey.message.internal.MessageBodyProviderNotFoundException: MessageBodyReader not found for media type=application/json